holmes/plugins/toolsets/infrainsights/redis_toolset.py

```
@@ -0,0 +1,534 @@
+        return f"Get Redis memory usage for instance: {instance_name}"
+
+
+class InfraInsightsRedisToolset(BaseInfraInsightsToolset):
+    """Redis toolset for InfraInsights"""
+    
+    def __init__(self, config: Optional[Dict[str, Any]] = None):
+        super().__init__(config)
+        
+        self.name = "InfraInsights Redis"
+        self.description = "Tools for investigating Redis instances, keys, and performance in InfraInsights"
+        self.tags = [ToolsetTag.CLUSTER]
```

Ruff (0.12.2)

Undefined name `ToolsetTag`

(F821)